src/read.c

```diff
@@ -2790,7 +2813,15 @@ static avifResult avifParseItemPropertyAssociation(avifMeta * meta, const uint8_
                         "a1op",
 
                         // HEIF: Section 6.5.11.1: "essential shall be equal to 1 for an 'lsel' item property."
-                        "lsel"
+                        "lsel",
+
+                        // MIAF: Section 7.3.9: "All transformative properties associated with coded and derived
+                        //                      images shall be marked as essential"
```

Comment:
We probably should add the version/edition number "2019/Amd. 2:2021" after "MIAF". Note the amendment 2.

In the original ISO/IEC 23000-22:2019 Section 7.3.9, this reads:

> All transformative properties associated with coded and derived images required or conditionally required by this document shall be marked as essential